app/gzac/src/main/kotlin/com/ritense/gzac/fvm/DummyFormViewModelLoader.kt

```diff
@@ -0,0 +1,20 @@
+package com.ritense.gzac.fvm
+
+import com.ritense.formviewmodel.viewmodel.ViewModelLoader
+import com.ritense.processlink.domain.ProcessLink
+import com.ritense.valtimo.camunda.domain.CamundaTask
+import org.springframework.core.Ordered
+import org.springframework.core.annotation.Order
+import org.springframework.stereotype.Component
+
+@Component
+@Order(Ordered.LOWEST_PRECEDENCE)
+class DummyFormViewModelLoader(
```

Comment:
Maybe rename to `NoopFormViewModelLoader` to be more in line with other classes in Valtimo